### ISSUE TITLE
[Suggestion]: When the player has unspent statpoints, you could use a different color for the numbers, to indicate it. (blue would be colorblind friendly)

### ISSUE THREAD

**Issue by SattesKrokodil** [Globox1997/LevelZ#28]
![image](https://user-images.githubusercontent.com/20082712/133045274-bb8b68b1-1db1-4d97-bdc9-e3a6268ca73d.png)

**Comment by SattesKrokodil** [comment]
#2 add skill reset potions, hard to get - I would like to see nether star + fluff items. Once you use the item, it gives you the opportuniy to reset one skill point. In systems like this, "mistakes" snowball into lategame and can make you feel really bad about your decisions later on. And without any ingame way to reset, people on servers will cry 100%   (=

**Comment by SattesKrokodil** [comment]
3. Mod compatibility.

example: 
Strength stat, it currently unlocks swords, stone iron etc.

suggestion: add tiers instead of certain weapon types, so other mods and modpacks can have an easy life adding compatibility on their own. Instead of "iron sword" it would be "melee weapons tier I"

**Comment by Globox1997** [comment]
Great ideas :D
Added first suggestion with [https://github.com/Globox1997/LevelZ/commit/9dd9bdaf7099f347136fe193d4bf264f2a93f7fa](https://github.com/Globox1997/LevelZ/commit/9dd9bdaf7099f347136fe193d4bf264f2a93f7fa).
Added the second suggestion: a potion called "Strange Potion" which resets a random skill and is brew able with dragon breath + netherstar, with [https://github.com/Globox1997/LevelZ/commit/50c53bdc104b36183dc97ac053519a4bad33020b](https://github.com/Globox1997/LevelZ/commit/50c53bdc104b36183dc97ac053519a4bad33020b)

**Comment by Zano1999** [comment]
Add experience gain separately for each skill like in other rpg games.
Example: Mining blocks increase mining skill, kill monsters increase strenght skill, etc.

**Comment by Monster-Zer0** [comment]
> 
> 
> Add experience gain separately for each skill like in other rpg games. Example: Mining blocks increase mining skill, kill monsters increase strenght skill, etc.

I feel like this would just cause a positive feedback loop unless experience gained decreases when level gets higher.

**Comment by Zano1999** [comment]
Of course, more and more experience should be required as you level up.  However, this progression system would make more sense than gaining experience (for example by fishing) and being able to choose to level up in strength as you can currently do.  Furthermore, the progression system I recommend would encourage you to devote yourself to various activities instead of the one that produces the most exp.

**Comment by DistantDig** [comment]
Honestly i like that "do the skill to level the skill" idea. If you think about it, its just like the jobs addon for levelz, except its kinda between the two. Even if it had to be its own mod instead of just a config option, i would still love to be able to level up that way instead
